bin/alluxio-mount.sh

```diff
@@ -87,6 +87,65 @@ function umount_ramfs_linux() {
   fi
 }
 
+function check_space_freebsd() {
+  local total_mem=$(($(sysctl -h hw.usermem | awk 'NR==1{print $2}' | sed 's/,//g')))
```

Comment:
Just trying to understand this command

> `sysctl -h hw.usermem`

prints out the system memory in human-readable format because of the `-h` flag

> `awk 'NR==1{print $2}`

This will print out the 2nd column from the first line, where the column is defined by a <space> char. I'm assuming the 2nd column is the value for the `hw.usermem`

> `sed 's/,//g'`

Removes an commas from the resulting output.

My only question with this set of commands is the `-h` flag. It will print in human readable format, but does that mean it will convert bytes to kb, mb, gb, etc dynamically based on the actual value?

In our linux version we use `/proc/meminfo` which always prints in kb, and then [multiply that value by 1024](https://github.com/Alluxio/alluxio/blob/a98d4f9bef7a12c2f89f1011101d4cbdbc72ecd8/bin/alluxio-mount.sh#L32). If we leave off the `-h` will it just give us the raw value in bytes? I think that is what we want.

(sorry I don't have a freebsd system available to test this myself)

Reply:
You're right, this was really sloppy from me. The correct way of doing it is `sysctl -n hw.usermem`. That way we can omit the awk and sed commands. I will update the PR.